### DISCUSSION TITLE
Ffat as audioSource

### DISCUSSION THREAD

**Discussion by bool-shark** [pschatzmann/arduino-audio-tools#638] [Q&A] [answered]
Hi Phil, maybe I'm missing it but not seeing an example with ffat for audioSource.

**Comment by pschatzmann** [comment]
I am not sure what you mean: ffat. 
What should this be ?

**Reply by bool-shark** [reply]
FATFS

**Comment by pschatzmann** [comment, marked as answer]
I am already supporting LitteFS and SPIFFS because somebody was paying for it. But I did not really see the added value.
Fee free to extend the functionality: It should be trivial.

**Reply by bool-shark** [reply]
.
